Doc/library/pickle.rst

```diff
@@ -511,6 +511,35 @@ The :mod:`pickle` module exports three classes, :class:`Pickler`,
       Release the underlying buffer exposed by the PickleBuffer object.
 
 
+.. _pickle-cli:
+
+Command-line interface
+----------------------
+
+The :mod:`pickle` module can be invoked as a script from the command line,
+it will display contents of the pickle files. However, when the pickle file
+that you want to examine comes from an untrusted source, ``-m pickletools``
+is a safer option because it does not execute pickle bytecode, see
+:ref:`pickletools CLI usage <pickletools-cli>`.
+
+.. code-block:: bash
+
+   python -m pickle pickle_file [pickle_file ...]
+
+The following option is accepted:
+
+.. program:: pickle
+
+.. option:: pickle_file
+
+   A pickle file, which can be a regular filename or a filepath.
+   Additionally, you can use ``-`` to read from standard input (stdin).
+   For example, you can pipe a pickle file to the script using a command like:
+
+   .. code-block:: bash
+      echo ... | python -m pickle -
```

Comment:
```suggestion
   A pickle file to read or ``-`` to indicate reading from standard input.
```

I don't think we need to explain what it means to read from stdin. Also, if we are to include the example, I think a working example would be preferred, but it's not a good idea to use an `echo` of a pickled content, so instead, let's just be a bit less precise. Note that it can (I think) be used as an interactive way as well.